README.md

```
@@ -1,438 +1,97 @@
-## Data Sources
+## Contributing
 
-- **OpenAQ**: Global air quality data from government monitoring stations
-- **JARTIC**: Japan Road Traffic Information Center - free historical traffic data for Japan
+See [CONTRIBUTING.md](CONTRIBUTING.md) for development guidelines.
 
 ## License
 
-MIT
\ No newline at end of file
+MIT License - see [LICENSE](LICENSE) for details.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)